.github/workflows/ci.yml

```diff
@@ -3,9 +3,24 @@
 name: WildFly Quickstarts CI
 
 on:
+  push:
+    branches:
+      - 'main'
```

Comment:
The workflow should be "branch abstract", to work on the NN.x branches without any modifications. This is why all dependency checkouts use ref: ${{ github.base_ref }}

Reply:
That only kind of works. What I mean by that, is that each branch will have it's own CI. Once you branch, e.g. 29.x, then that branch runs off it's own workflow, not the one from main.

That said, this change is not needed at this point really. It's really to get around dependabot jobs from running two different CI runs. That said, there are other ways to handle that.